source/github/access.go

```diff
@@ -87,7 +87,7 @@ func (c *client) TeamAccess(u *library.User, org, team string) (string, error) {
 	// iterate through each element in the teams
 	for _, t := range teams {
 		// skip the team if does not match the team we are checking
-		if !strings.EqualFold(team, t.GetName()) {
+		if !strings.EqualFold(team, strings.ToLower(t.GetName())) {
```

Comment:
Do we need this `strings.ToLower()`?

AFAIK, `strings.EqualFold()` implicitly handles case-insensitivity 👍 

So, I don't believe we have to worry about upper or lower casing the team name:

https://pkg.go.dev/strings#EqualFold

And a sample from Go Playground:

https://play.golang.org/p/5qGWfV6yzap